### PR TITLE
Add helpful hints about setting GUROBI_HOME environment variable. Use…

### PR DESCRIPTION
**Pull request by r-barnes**
…ful for folks who might add the package without checking the installation instructions.

### CODE REVIEW

**Comment by odow** [comment]
How about including a link to the README so people can read the full instructions?

**Comment by r-barnes** [comment]
Hmm, to `https://github.com/JuliaOpt/Gurobi.jl/blob/master/README.md` or `https://github.com/JuliaOpt/Gurobi.jl`?

**Comment by odow** [comment]
Probably just include something like `For more information go to https://github.com/JuliaOpt/Gurobi.jl.`

File: deps/build.jl [hunk 1]
```diff
@@ -59,5 +59,5 @@ for l in paths_to_try
 end
 
 if !found && !haskey(ENV, "GUROBI_JL_SKIP_LIB_CHECK")
-    error("Unable to locate Gurobi installation. Note that this must be downloaded separately from gurobi.com")
+    error("Unable to locate Gurobi installation. Note that this must be downloaded separately from gurobi.com. https://github.com/JuliaOpt/Gurobi.jl")
```
Comment (odow) [inline deps/build.jl]: You missed the "For more information" bit here

**Comment by odow** [comment]
Thanks!

**Pull request merged**